[No QA] Add PR template to this library (#58)
Expensify/react-native-onyx: .github/PULL_REQUEST_TEMPLATE.md

<If necessary, assign reviewers that know the area or changes well. Feel free to tag any additional reviewers you see fit.>

### Details
<Explanation of the change or anything fishy that is going on>

### Related Issues
<Please replace GH_LINK with the link to the GitHub issue this Pull Request is related to>
GH_LINK

### Automated Tests
<!---

> Most changes to Onyx should have an accompanying tests. Describe the tests you added or if no tests were added an explanation about why one was not needed.

Reviewer: should be `have accompanying tests` or `have an accompanying test`

--->

### Linked PRs
<!---

> Please include links to any PRs where this change will be implemented.

Reviewer 2: ```suggestion
Please include links to any PRs where this change will start being used
```
?

Author: It does sound a little weird maybe

>Please include links to any PRs in repos that update the `package.json` version.
